src/pages/r/[communityId]/submit.tsx

```
@@ -0,0 +1,34 @@
+type submitProps = {};
+
+const SubmitPostPage: React.FC<submitProps> = () => {
+  const [user] = useAuthState(auth);
+  const { communityStateValue } = useCommunityData();
+  return (
+    <SubmitPage>
+      <TitleBox>
+        <NorText>Create a Post</NorText>
+      </TitleBox>
+      <PageContent>
+        <>{user && <NewPostForm user={user} />}</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)